src/plugins/intel_cpu/src/nodes/generate_proposals.cpp

```diff
@@ -306,6 +188,12 @@ GenerateProposals::GenerateProposals
     coordinates_offset_ = proposalAttrs.normalized ? 0.f : 1.f;
 
     roi_indices_.resize(post_nms_topn_);
+    try {
+        nms_kernel_ = compileNMSKernel();
+    } catch (const InferenceEngine::GeneralError &e) {
+        DEBUG_LOG("Unable to create JIT version of GenerateProposals: ",
```

Comment:
Instead of throwing exception you could just return empty `std::unique_ptr`

Reply:
`compileNMSKernel()` method was removed in sake of `createPrimitive()` implementation.